docs/references/commands.txt

```diff
@@ -0,0 +1,326 @@
+   in the uaclient/messages folder.
+
+Available commands
+==================
+
+The currently available commands are:
+
+- `pro api`_
+- `pro attach`_
+- `pro auto-attach`_
+- `pro collect-logs`_
+- `pro config`_
+- `pro detach`_
+- `pro disable`_
+- `pro enable`_
+- `pro fix`_
+- `pro help`_
+- `pro refresh`_
+- `pro security-status`_
+- `pro status`_
+- `pro system`_
+
+
+pro api
+=======
+
+**Usage:**
+
+``pro api [-h] [--show-progress] [--args [OPTIONS ...]] [--data DATA] endpoint``
+
+**Description:**
+
+Calls the Client API endpoints.
+
+For a list of all of the supported endpoints and their structure,
+please refer to the Pro Client API reference guide:
+
+https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/references/api/
+
+pro attach
+==========
+
+**Usage:**
+
+``pro attach [-h] [--no-auto-enable] [--attach-config ATTACH_CONFIG] [--format {cli,json}] [token]``
+
+**Description:**
+
+Attach this machine to an Ubuntu Pro subscription with a token obtained from:
+https://ubuntu.com/pro/dashboard
+
+When running this command without a token, it will generate a short code
+and prompt you to attach the machine to your Ubuntu Pro account using
+a web browser.
+
+The --attach-config option can be used to provide a file with the token
+and optionally, a list of services to enable after attaching. To know more,
+visit:
+https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/howtoguides/how_to_attach_with_config_file/
+
+The exit code will be:
+
+    * 0: on successful attach
+    * 1: in case of any error while trying to attach
+    * 2: if the machine is already attached
+
+pro auto-attach
+===============
+
+**Usage:**
+
+``pro auto-attach [-h]``
+
+**Description:**
+
+Automatically attach on an Ubuntu Pro cloud instance.
+
+pro collect-logs
+================
+
+**Usage:**
+
+``pro collect-logs [-h] [-o OUTPUT]``
+
+**Description:**
+
+Collect logs and relevant system information into a tarball.
+This information can be later used for triaging/debugging issues.
+
+pro config
+==========
+
+**Usage:**
+
+``pro config [-h] {show,set,unset} ...``
+
+**Description:**
+
+Manage Ubuntu Pro Client configuration on this machine
+
+pro config show
+---------------
+
+**Usage:**
+
+``pro config show [-h] [key]``
+
+**Description:**
+
+Show customizable configuration settings
+
+pro config set
+--------------
+
+**Usage:**
+
+``pro config set [-h] key_value_pair``
+
+**Description:**
+
+Set and apply Ubuntu Pro configuration settings
+
+pro config unset
+----------------
+
+**Usage:**
+
+``pro config unset [-h] key``
+
+**Description:**
+
+Unset an Ubuntu Pro configuration setting, restoring the default value.
+
+pro detach
+==========
+
+**Usage:**
+
+``pro detach [-h] [--assume-yes] [--format {cli,json}]``
+
+**Description:**
+
+Detach this machine from an Ubuntu Pro subscription.
+
+pro disable
+===========
+
+**Usage:**
+
+``pro disable [-h] [--assume-yes] [--format {cli,json}] [--purge] service [service ...]``
```

Comment:
Probably an obvious question, but can you disable more than one service at a time this way?

Reply:
yes - this is represented by the `[service ...]` in the `service` option. We should say one or more right. Changing it.